### DISCUSSION TITLE
How to host Postgres & Redis on same server?

### DISCUSSION THREAD

**Discussion by mariusa** [caprover/caprover#1700] [Q&A] [answered]
Hi, I see that Postgres & Redis are available as one-click apps, as separate servers.

How could one run Postgres & Redis on same server? Reason: save overhead resources (running the OS) on VPS.
Also, it's a good practice for web apps connecting to a DB to have same caching (using Redis).

Thanks!

**Comment by githubsaturn** [comment, marked as answer]
>  as separate servers.

This is incorrect. They are just separate containers.
